tools/stats/scribe.py

```diff
@@ -2,6 +2,21 @@
 import bz2
 import os
 import json
+from typing import Dict, Any, List, Union
+
+
+_lambda_client = None
+
+
+def aws_lambda() -> Any:
+    global _lambda_client
+    # lazy import so that we don't need to introduce extra dependencies
+    import boto3  # type: ignore[import]
+
+    if _lambda_client is None:
+        _lambda_client = boto3.client("lambda")
+
+    return _lambda_client
```

Comment:
How about `def aws_lambda_invoke(func_name: str, input: List[Dict[str, Any]]) -> Any` function that calls creates client and encodes input and decodes output?